apps/schematic/api/schematic_api/test/test_synapse_endpoints.py

```diff
@@ -330,3 +339,22 @@ def test_get_project_manifest_metadata_page(self) -> None:
         for manifest in response.json["manifests"]:
             assert isinstance(manifest, dict)
             assert list(manifest.keys()) == MANIFEST_METADATA_KEYS
+
+
+@pytest.mark.synapse
+@pytest.mark.secrets
+class TestPurgeSynapseCache:  # pylint: disable=too-few-public-methods
+    """Tests purge_synapse_cache"""
+
+    def test_success(self) -> None:
+        """Tests for a successful purge"""
+        synapse_store = SynapseStorage(
+            access_token=SYNAPSE_TOKEN, synapse_cache_path=AWS_SYNAPSE_CACHE_PATH
```

Comment:
if when running the api locally, we are also using AWS_SYNAPSE_CACHE_PATH as cache path, I think the test should be run using a different cache path.. otherwise it will clear all the cache every time the test gets run localy..